src/DynamoCore/BuiltInAndOperators/BuiltIn.xml

```diff
@@ -157,9 +157,10 @@
     </member>
     <member name="LoopWhile">
       <summary>Constructs a while loop node given an initial value, a condition and a loop body.</summary>
-      <param name="init">initial value</param>
+      <param name="initialValue">initial value for loop</param>
       <param name="continueWhile">while loop condition</param>
       <param name="loopBody">while loop body</param>
+      <returns name="result">result from loop</returns>
```

Comment:
`var[]..[]` type in description?

Reply:
![image](https://user-images.githubusercontent.com/25138291/93858744-9279f380-fcb4-11ea-9d52-60b50affe5db.png)

@aparajit-pratap  do you know if LoopWhile works for lists? I wasn't able to make it work, if it doesn't we should rename the output description to: 
`type: number (result from loop)`

Reply:
Addressed in 4dc47fd